src/api/spec/lib/backend/api/cloud_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe Backend::Api::Cloud do
+  context '#upload' do
+    let(:user) { create(:confirmed_user) }
+    let!(:ec2_configuration) { create(:ec2_configuration, user: user) }
+
+    it 'crafts a correct backend request' do
+      # rubocop:disable RSpec/MessageSpies
```

Comment:
Why do we need to disable the Rubocop?

Why are we not switching to ```EnforcedStyle: receive``` then?
http://www.rubydoc.info/gems/rubocop-rspec/1.10.0/RuboCop/Cop/RSpec/MessageSpies

Reply:
I wrote it in my commit message. Essentially, because I would have to configure the backend test suite (have_received doesn't stub the request). And since I am don't intent to test the backend connection with this test, I use receive.